### PR TITLE
Form controls errors

### LINKED ISSUE #1108: Field-set class - no attributes - ??
Continue to refactor the generator I'm working on.

Was generalizing the introduction text for inputs and field sets. (Labels for text inputs and areas and drop-downs, field sets for radio, checkbox, and date input.)

Ran across this:

For radio and checkboxes there is a class on the <fieldset>: class="usa-fieldset-inputs isa-sans"

For date input - that class does not exist.

When I perform inspect element on the radio and checkboxes to see what attributes are assigned to class - I don't see any. 

Orphaned class? Or, a possible discrepancy between the sass generated here versus from NPM?

Just curious.

### PR DESCRIPTION
**Pull request by joshbruce**
Fixes #1108

Date input is a little tricky due to legend default styling. Could remove the first variation in favor of a consistent typography across all label-like concepts. Without the field-set class, the default is used, with it, the legend takes on the properties of a label.

### CODE REVIEW

File: docs/_elements/form-controls.md [hunk 1]
```diff
@@ -281,6 +388,49 @@ lead: Form controls allow users to enter information into a page.
       </div>
     </div>
   </fieldset>
+
+  <fieldset class="usa-fieldset-inputs">
+    <legend>Date of birth</legend>
+    <span class="usa-form-hint usa-datefield-hint" id="dobHint">For example: 04 28 1986</span>
+
```
Comment (jbrucegsa) [inline docs/_elements/form-controls.md]: Date input is a little tricky due to legend default styling. Could remove the first variation in favor of a consistent typography across all label-like concepts. Without the field-set class, the default is used, with it, the legend takes on the properties of a label.

**Comment by jbrucegsa** [comment]
ps. I wasn't sure if I needed to update the SCSS in the /src; so, I erred on the side of caution. Please let me know if that will be necessary in the future.

**Comment by jbrucegsa** [comment]
Fixes #1076

**Comment by maya** [comment]
Noting that this PR includes design changes and we haven't finished creating our design contributor guidelines cc @carodew

**Comment by jbrucegsa** [comment]
Fixes #1094

@maya - Thanks. May also want @msecret (no autocomplete on my phone) per the initial issue fix.

**Comment by bradnunnally** [comment]
This solution looks good to me! Assigning to @juliaelman for closing out.

**Comment by maya** [comment]
@bradnunnally since this is visual design-related, shouldn't vizdee review it first?

**Comment by bradnunnally** [comment]
Assigned to @ericadeahl for visual design review.

**Comment by bradnunnally** [comment]
After team discussion, we are going to close this PR and tie it into issue #860. Thanks for all the input and we will use it as guidance when revisiting the form elements overall design.

**Pull request closed without merging**